packages/dds/tree/src/simple-tree/nodeKeys.ts

```diff
@@ -3,9 +3,16 @@
  * Licensed under the MIT License.
  */
 
-import { LocalNodeKey, NodeKeyIndex, NodeKeyManager, StableNodeKey } from "../node-key/index.js";
+import {
+	LocalNodeKey,
+	NodeKeyIndex,
+	NodeKeyManager,
+	StableNodeKey,
+	// eslint-disable-next-line import/no-internal-modules
+} from "../feature-libraries/node-key/index.js";
 
-import { FlexTreeObjectNode } from "./flexTreeTypes.js";
+// eslint-disable-next-line import/no-internal-modules
+import { FlexTreeObjectNode } from "../feature-libraries/flex-tree/flexTreeTypes.js";
```

Comment:
If this is going to be at the simple tree layer, you shouldn't map to flex tree nodes. Prabably use TreeNode instead.

Reply:
Updated to use `TreeNode`